css/desktop-styling.css

```diff
@@ -0,0 +1,771 @@
+    order: 1;
+  }
+
+  button {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px;
+    gap: 10px;
+    width: 162px;
+    height: 48px;
+    border: 1px solid #6070ff;
+    border-radius: 8px;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+  }
+
+  .social-about {
+    padding: 20px 0;
+    margin-left: 10px;
+    margin-top: 15px;
+    align-self: flex-start;
+    margin-bottom: 5%;
+  }
+
+  .connection-about {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    color: #7f8cff;
+  }
+
+  .media-icons-about {
+    display: flex;
+    list-style: none;
+    padding: 5px 0;
+    gap: 10px;
+    left: 28.89%;
+    right: 70.13%;
+    top: 12.53%;
+    bottom: 87.08%;
+    color: #505f79;
+  }
+
+  .about-submit-btn {
+    width: 200px;
+    height: 50px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    margin-top: 15px;
+    padding: 10px;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #396df2;
+    margin-left: 10px;
+  }
+
+  .desktop-image4 {
+    padding: 10px;
+    margin-top: 5%;
+  }
+
+  .lang {
+    display: flex;
+    flex-direction: column;
+    margin-top: 5%;
+    padding: 10px;
+  }
+
+  .about-lang {
+    height: 24px;
+    left: 24px;
+    right: 78px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+    display: flex;
+    align-items: center;
+    color: #344563;
+  }
+
+  .seperator-btn {
+    width: 450px;
+    margin-top: 5px;
+  }
+
+  .display-flex ul {
+    display: flex;
+    list-style: none;
+    gap: 10px;
+    padding: 10px;
+  }
+
+  .language {
+    width: 98px;
+    height: 24px;
+    left: 12px;
+    margin-top: 20px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    display: flex;
+    align-items: center;
+    color: #253858;
+  }
+
+  .language-list {
+    width: 122px;
+    height: 120px;
+    background: #f7f7f9;
+    border-radius: 8px;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    margin-top: 50px;
+  }
+
+  .frame-work {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+    display: flex;
+    align-items: center;
+    color: #344563;
+    padding: 20px;
+  }
+
+  .works-frame {
+    margin-top: 10px;
+    padding: 30px;
+  }
+
+  .down-arrow {
+    right: 33.33%;
+    top: 25%;
+    bottom: 25%;
+    margin-left: 75%;
+    left: 50%;
+  }
+
+  .left-arrow {
+    padding: 0 60%;
+  }
+
+  .left-arrow2 {
+    padding: 0 75%;
+  }
+
+  .contanct-containter {
+    margin-top: 10%;
+    display: flex;
+    background-color: #6070ff;
+    border-top-left-radius: 100px;
+    position: relative;
+  }
```

Comment:
- [ ] As suggested by the previous reviewer, you should stick to the figma design as close as possible. Kindly consider checking the width of the container of both the text and the form on the figma design and use the width on your design so as to produce a close replica of the guiding template. Take look at both designs and walk your design alongside the figma design.

### Your Design Desktop
![your-form](https://user-images.githubusercontent.com/26597432/206667783-e6e61e71-69b3-4c09-b7f1-aff6a1265b0d.png)

### Figma Design Desktop
![figma-form](https://user-images.githubusercontent.com/26597432/206667807-5c378f8a-a19a-4f5d-8dfb-8c3d8f0dd1b7.png)

Reply:
Hello @Qoosim thank you so much for the feedback, okay kindly what am I supposed to do, reduce the size?? Sincein my machine it's showing the same results as figma...kindly what exactly am I supposed to do here
![Screenshot from 2022-12-09 13-06-03](https://user-images.githubusercontent.com/55842414/206677412-7bccce30-2061-44ed-98c6-c72788a0a909.png)

Reply:
![Screenshot from 2022-12-09 13-07-58](https://user-images.githubusercontent.com/55842414/206677793-dae04032-e927-411d-910b-d06f4e8e873d.png)

Reply:
- I will change the border-radius,more details on what else I should change will be appreciated. Thank you already.